### ISSUE TITLE
"sorry, could not start: /path/to/application"

### ISSUE THREAD

**Issue by Lew-Rockwell-Fan** [nomacs/nomacs#592]
**Describe the bug**
Applications added to the 'open with' menu don't start. All that happens is a brief error message printed out on the image like in the title.  I've added desktop and direct links to executables. This is in pclos, openbox. I ran Nomacs in a terminal and didn't see any other error message, just:
""Sorry, I could not start: /usr/share/applications/feh.desktop"

**Comment by leejuyuu** [comment]
Try using `/usr/bin/feh` instead. This works for me.

**Comment by leejuyuu** [comment]
Closing because of lack of feedback. Feel free to reopen if needed.

**Comment by Lew-Rockwell-Fan** [comment]
Wow! A response after 4 years. By that time I had long since given up on Nomacs and switched to Viewnior, which is why "lack of feedback," LOL. But I'm trying it again under Lubuntu 24.04, still in a plain Openbox DE, and the problem persists, but, yes, linking directly to the file works. But only until I close the window. When I open nomacs again, the the new entry is gone. But that's really a different issue, so I'll file a separate report. Thanks.

**Comment by scrubbbbs** [comment]
Yes this is annoying and has tripped me up a few times. But there is no standard way to launch apps from `.desktop` files, each desktop environment wants to do it's own thing. I think we should try glib's `GDesktopAppInfo` which should be fine for the majority and we just won't support the others. We could load glib dynamically so we aren't adding another compile-time dependency as well.
